trajectory_interval: 0.1
takeoff_height: 5.0